bouffalo-hal/src/sdio.rs

```diff
@@ -234,6 +240,8 @@ pub enum AutoCMDMode {
     CMD12 = 1,
     /// Auto CMD23 Enable.
     CMD23 = 2,
+    /// Set this bit to zero.
+    ZERO = 0,
```

Comment:
```suggestion
    /// Set this bit to zero.
    None = 0,
```
Should we consider using `None` to indicate that no automatic command is enabled?